src/coreclr/src/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -12,10 +12,46 @@
     <OutputPath>$(BinDir)/ilc</OutputPath>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
-    <RuntimeIdentifiers>linux-x64;linux-musl-x64;win-x64</RuntimeIdentifiers>
+    <RuntimeIdentifiers>linux-x64;win-x64</RuntimeIdentifiers>
```

Comment:
Had to delete MUSL entry because we don't have an objwriter package for that and NuGet insists on restoring the packages for every runtime under the sun, even though we don't build for it.